fiduceo_uncertainties.f90

```diff
@@ -1446,9 +1471,6 @@ SUBROUTINE radiance_uncertainties(i,outData,coefs1,coefs2,coefs3,FCDR,&
        FCDR%ucict3 = NAN_R
        FCDR%ucict4 = NAN_R
        FCDR%ucict5 = NAN_R
-       FCDR%ucs3 = NAN_R
```

Comment:
And again?

Reply:
Yes sorry. I have put back in the following:

-->
       FCDR%ucs3 = NAN_R
       FCDR%ucs4 = NAN_R
       FCDR%ucs5 = NAN_R